CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # OST Wallet SDK Changelog
 
+## Version 2.3.7
+### Bug Fix:
+* In recover device flow added propagation of OstError.
```

Comment:
Inaccurate error is thrown when application runs out of memory during recover device workflow.